string source = rogr
string cameratype
string property
time timestamp



